frontend/www/js/omegaup/components/arena/ClarificationList.vue

```diff
@@ -0,0 +1,153 @@
+        role="dialog"
+        aria-hidden="true"
+      >
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title">{{ T.wordsNewClarification }}</h5>
+              <button
+                type="button"
+                class="close w-auto"
+                data-dismiss="modal"
+                aria-label="Close"
+              >
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body">
+              <label>
+                {{ T.wordsProblem }}
+                <select v-model="newClarification.problemAlias">
+                  <option
+                    v-for="problem in contestProblems"
+                    v-bind:value="problem.alias"
+                  >
+                    {{ problem.title }}</option
+                  >
+                </select>
+              </label>
+              <textarea
+                class="w-100"
+                maxlength="200"
+                v-model="newClarification.message"
+                v-bind:placeholder="T.arenaClarificationCreateMaxLength"
+              ></textarea>
+            </div>
+            <div class="modal-footer">
+              <button
+                type="button"
+                class="btn btn-secondary"
+                data-dismiss="modal"
+              >
+                {{ T.wordsClose }}
+              </button>
+              <button
+                type="button"
+                class="btn btn-primary"
+                v-on:click.prevent="sendClarification"
+              >
+                {{ T.wordsSend }}
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="table-responsive">
+      <table class="table mb-0">
+        <thead>
+          <tr>
+            <th class="text-center" scope="col" v-if="inContest">
+              {{ T.wordsContest }}
+            </th>
+            <th class="text-center" scope="col" v-else="">
+              {{ T.wordsProblem }}
+            </th>
+            <th class="text-center" scope="col">{{ T.wordsAuthor }}</th>
+            <th class="text-center" scope="col">{{ T.wordsTime }}</th>
+            <th class="text-center" scope="col">{{ T.wordsMessage }}</th>
+            <th class="text-center" scope="col">{{ T.wordsResult }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <omegaup-clarification
+            v-for="clarification in clarifications"
+            v-bind:in-contest="inContest"
+            v-bind:key="clarification.clarification_id"
+            v-bind:clarification="clarification"
+          ></omegaup-clarification>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+/* Deep allows child components to inherit the styles (see: https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors) */
```

Comment:
nit:

```suggestion
// Deep allows child components to inherit the styles (see: https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors)
```

(no nos gustan los block-comments en general)

Reply:
CSS no entiende `//`. Lo pasaré a `.scss`